siga/src/main/java/br/gov/jfrj/siga/vraptor/DpPessoaController.java

```diff
@@ -166,13 +166,13 @@ protected DpPessoaDaoFiltro createDaoFiltro() {
 		String buscarFechadas = param("buscarFechadas");
 		flt.setBuscarFechadas(buscarFechadas != null ? Boolean.valueOf(buscarFechadas) : false);
 		String buscarSubstitutos = param("buscarSubstitutos");
-		if (buscarSubstitutos != null) {
-			flt.setBuscarSubstitutos(true);
-			flt.setId(getTitular().getId());
-		} else {
-			flt.setBuscarSubstitutos(false);
-		}
-		
+//		if (buscarSubstitutos.equals(true)) {
+//			flt.setBuscarSubstitutos(true);
+//			flt.setId(getTitular().getId());
+//		} else {
+//			flt.setBuscarSubstitutos(false);
+//		}
+//		
 		flt.setSituacaoFuncionalPessoa("");
```

Comment:
Penso que esta solução não seja adequada. Além disso, pode impactar na PR #206.

Reply:
Corrigir a condição não seria o suficiente?
Algo assim:

```suggestion
if (buscarSubstitutos != null &&  Boolean.valueOf(buscarSubstitutos))) {
    flt.setBuscarSubstitutos(true);
    flt.setId(getTitular().getId());
} else {
    flt.setBuscarSubstitutos(false);
}
		
```

Reply:
Isso mesmo, @caiosantos-codata. Vou verificar o problema hoje.
De último caso a gente reverte o commit e deixa o problema até que possa ser desrevertido novamente com a solução definitiva.

Reply:
ok, @michelrisucci-codata . 

penso que ainda vale a pena uma nova verificação antes de fazer um revert. 
posso verificar também.